Week-2/Exercise2.1/css/style.css

```diff
@@ -0,0 +1,269 @@
+:root {
+  --font-color: #4d3232;
```

Comment:
Lets have the color variables representing what color they are and not where its applicable to. 

```
  --font-color: #4d3232;
  --grey-color: #f1f1f1;
  --footer-color: #6495ed;
  --white-color: #ffffff;
  --link-color: #1358d0;
  --link-hover-color: #ff0000;
  --header-color: #003c68;
  --form-color: #b1c4c5;
  --button-color: #374151;
  --button-hover-color: #009999;
```

Instead of the above lets do the below

```
  --black-color: #000;
  --grey-color: #f1f1f1;
  --blue:color: #6495ed;
  --white-color: #ffffff;
  --primary-grey-color: #1358d0;
  --secondary-grey-color: #ff0000;
```